cpmpy/model.py

```diff
@@ -231,4 +231,18 @@ def from_file(fname):
         """
         with open(fname, "rb") as f:
             return pickle.load(f)
-    
+
+    def copy(self, memodict={}):
+        """
+            Copies a the model to a new instance.
```

Comment:
Returns a copy of the model?

something about the 'memodict' argument too.

and something about it being a shallow copy or deep copy?

Reply:
I was thinking about shallow/deep copy distinctions as well. But I can see no reason why someone would need a shallow copy of a model.
As variables are still shared between models then, solving one model will affect the variables in the other one.
Might still be usefull to clarify that it is a deep copy anyway indeed